qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +66,27 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        obs = self.pm.importable_formats
+        exp = {
+            'IntSequenceDirectoryFormat': IntSequenceDirectoryFormat,
+            'MappingDirectoryFormat': MappingDirectoryFormat,
+            'IntSequenceV2DirectoryFormat': IntSequenceV2DirectoryFormat,
+            'IntSequenceFormatV2': IntSequenceFormatV2,
+            'FourIntsDirectoryFormat': FourIntsDirectoryFormat,
+            'IntSequenceFormat': IntSequenceFormat
+        }
+
+        self.assertEqual(obs, exp)
+
+    def test_importable_formats_with_dummy(self):
+        exp = "DummyFormat"
+        obs = self.pm.importable_formats.keys()
+        self.assertFalse(exp in obs)
+
+        obs = [item[0] for item in self.plugin.formats.items()]
```

Comment:
It'd be better to use `self.pm.formats` (i.e. the `PluginManager`) in case there are other "dummy" plugins defined in the framework for testing.

The test case can be simplified to:

```python
self.assertTrue('DummyFormat' in self.pm.formats)
```